src/components/MultiLink.vue

```diff
@@ -241,23 +249,29 @@ export default Vue.extend({
     },
 
     dragNode(node: Node, event: MouseEvent) {
-      event.preventDefault();
+      if (!(this.$refs.svg instanceof Element)) { throw new Error('SVG is not of type Element'); }
```

Comment:
You may want to follow standard style here (break this into three lines instead of one long line).

Reply:
See 877d067